src/lsm/compaction.zig

```diff
@@ -82,6 +82,11 @@ pub fn CompactionType(
             IteratorB,
         );
 
+        const CompactionStrategy = enum {
+            table_move,
+            table_merge,
+        };
```

Comment:
This is great. Also lines up nicely.

Nit: Reorder the enums alphabetically, but also in the order of most frequent use. So `merge` before `move`. This is a bit safer if the enum is ever messed up, then we won't move but rather merge.

Reply:
We could also drop the `table_` prefixes here I think. It will read nicer than further down whenever we assert on `strategy`. Also because when we merge, we merge multiple tables.